utils/build/docker/nodejs/express4-typescript/iast.ts

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+import type { Express, Request, Response } from 'express'
+
+const crypto = require('crypto')
+
+module.exports = function (app: Express) {
+    app.get('/iast/weak_randomness/test_insecure', (req: Request, res: Response) => {
+        const randomNumber = Math.random()
+        res.send(`OK:${randomNumber}`)
+    })
+
+    app.get('/iast/weak_randomness/test_secure', (req: Request, res: Response) => {
+        const randomBytes = crypto.randomBytes(256).toString('hex')
+        res.send(`OK:${randomBytes}`)
+    })
+}
```

Comment:
whoops looks like we're lacking some linting here
```suggestion
module.exports = function (app: Express) {
  app.get('/iast/weak_randomness/test_insecure', (req: Request, res: Response) => {
    const randomNumber = Math.random()
    res.send(`OK:${randomNumber}`)
  })

  app.get('/iast/weak_randomness/test_secure', (req: Request, res: Response) => {
    const randomBytes = crypto.randomBytes(256).toString('hex')
    res.send(`OK:${randomBytes}`)
  })
}
```

Reply:
Future work: add lint task in typescript project